functionality_package_pickup.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+
+def package_pickup():
+    package_number = input("Proszę wpisać numer swojej paczki: ")
+    searching_package_in_file(package_number)
+
+
+def searching_package_in_file(package_number):
+    with open("parcel_list.json", "r") as f:
+        parcel_dict = json.load(f)
+
+        search_key = package_number
+
+        if search_key in parcel_dict:
```

Comment:
zrobiłbym to raczej pope, wtedy mamy taz dane paczki :) 
zmienna search_key jak dla mnie nie potrzebna
```suggestion
        package = parcel_dict.pop(package_number)
        if package:
              zapis do JSONA ktory juz masz
              ewentualne wyprintowanie info paczki ze zmiennej package
        else:
             print(f"Niestety w systemie nie odnajdujemy paczki o numerze {package_number}")
```